docs/rum_collection.md

```diff
@@ -0,0 +1,271 @@
+
+    {{< tabs >}}
+    {{% tab "US" %}}
+
+```swift
+Datadog.initialize(
+    appContext: .init(),
+    configuration: Datadog.Configuration
+        .builderUsing(
+            rumApplicationID: "<rum_application-id>",
+            clientToken: "<client_token>",
+            environment: "<environment_name>"
+        )
+        .set(serviceName: "app-name")
+        .build()
+)
+```
+
+    {{% /tab %}}
+    {{% tab "EU" %}}
+
+```swift
+Datadog.initialize(
+    appContext: .init(),
+    configuration: Datadog.Configuration
+        .builderUsing(
+            rumApplicationID: "<rum_application-id>",
+            clientToken: "<client_token>",
+            environment: "<environment_name>"
+        )
+        .set(serviceName: "app-name")
+        .set(rumEndpoint: .eu)
+        .build()
+)
+```
+
+    {{% /tab %}}
+    {{< /tabs >}}
+
+3. Configure and register the RUM Monitor. You only need to do it once, usually in your `AppDelegate` code:
+
+    ```swift
+    import Datadog
+
+    Global.rum = RUMMonitor.initialize()
+    ```
+
+The RUM SDK offers two instrumentation methods:
+
+- Auto-instrumentation (recommended) - the SDK tracks views, resources, actions and errors automatically.
+- Manual instrumentation - you instrument your code to send RUM events.
+
+**Note**: It is possible to mix both methods.
+
+## Auto Instrumentation
+
+### RUM Views
+
+To enable RUM views tracking, use the `.trackUIKitRUMViews(using:)` option when configuring the SDK:
+```swift
+Datadog.Configuration
+   .builderUsing(...)
+   .trackUIKitRUMViews(using: predicate)
+   .build()
+```
+
+The `predicate` must be a class or struct conforming to our `UIKitRUMViewsPredicate` protocol:
+```swift
+public protocol UIKitRUMViewsPredicate {
+    func rumView(for viewController: UIViewController) -> RUMViewFromPredicate?
+}
+```
+
+Inside the `rumView(for:)` implementation, your app should decide if a given `UIViewController` instance should start the RUM view or not (return `nil` in such case). The returned value of `RUMViewFromPredicate` should specify at least the `path` for created RUM view. Refer to code documentation comments for more details.
+
+**Note**: the SDK will call `rumView(for:)` many times while your app is running. Your implementation of the predicate should not depend on the order of SDK calls.
+
+### RUM Resources
+
+To enable RUM resources tracking, use the `.track(firstPartyHosts:)` option when configuring the SDK:
+```swift
+Datadog.Configuration
+   .builderUsing(...)
+   .track(firstPartyHosts: ["your.domain.com"])
+   .build()
+```
+Also, assign our `DDURLSessionDelegate()` as a `delegate` of the `URLSession` you want to monitor, e.g.:
```

Comment:
maybe we can state that if one of them isn't there, it won't work?

Reply:
IMO we should keep it short. This guide shows how to configure things to make it work, and we should not focus on explaining missconfiguration.